### PR TITLE
Adds watertight/container tags to a few items

### LINKED ISSUE #18569: Suggestion: Allow glasses and cups to hold liquid.
Glasses, ceramic bowls and cups cannot hold liquid. Hot tea can never know the embrace of a proper cup. Glasses, begging for a frosty one will never receive their desired frosty one.

### PR DESCRIPTION
**Pull request by MisterFox**
This change adds watertight/container tags to the following items so they can hold liquids:
- Pewter bowl - 1 unit
- Ceramic bowl - 1 unit
- Glass bowl - 1 unit
- Ceramic cup - 1 unit
- Glass (renamed) - 2 units ("tall glass")

It also renames 'glass' to 'glass cup' to maintain consistency.

Fixes #18569

### CODE REVIEW

**Comment by mugling** [comment]
Does this not result in you being able to put a glass of water in your rucksack?

**Comment by MisterFox** [comment]
I don't imagine it does - -  I assume the "resealable" flag is the one that allows that and I just added "watertight" so they can hold liquid when wielded or on the ground. I will confirm when I get back later.

**Comment by MisterFox** [comment]
@mugling No -- this does not allow you to put a glass of water in your inventory. It works as expected like other unsealable watertight containers.

File: data/json/items/containers.json [hunk 1]
```diff
@@ -80,6 +80,74 @@
...
+        "symbol": ",",
+        "color": "cyan",
+        "name": "glass cup",
+        "name_plural": "glasses",
+        "description": "A tall glass, just begging for a frosty one!",
+        "price": 400,
+        "price_postapoc": 0,
+        "material": "glass",
+        "weight": 375,
+        "volume": 1,
```
Comment (Rivet-the-Zombie) [inline data/json/items/containers.json]: The glasses' volume should go up to match their capacity.

**Comment by MisterFox** [comment]
It's not clear to me what the 'contain' quality means -- just that it can hold something inside it, or is there a specific condition? I was thinking about removing the quality from the creamic cup (teacup), but I saw that even the aluminum can had 'contain'.

**Comment by Coolthulhu** [comment]
There's a JSON error that needs correcting. Checkout the "Details".

> It's not clear to me what the 'contain' quality means -- just that it can hold something inside it, or is there a specific condition?

It is used for recipes that don't produce liquids, but need some container to mix stuff in. For example, sugar requires mixing lye, water and beets in a container, but produces something that doesn't need to be contained.

The quality was useful before buckets were a thing, nowadays it should probably be implicit, calculated from containing ability, rather than explicit.

**Comment by BorkBorkGoesTheCode** [comment]
Try adding contains 1 to Arcana's offering_chalice_used.

**Pull request merged**